proposals/array_modify.json

```diff
@@ -19,7 +19,7 @@
         },
         {
             "name": "values",
-            "description": "The values to fill the array with.",
+            "description": "The values to insert into the `data` array.",
```

Comment:
I've used fill here as insert into sounded a bit too much like "add", but "update" is also allowed. But if you see an update as insert & remove, it works...

Reply:
`fill` sounds like overwriting the whole array, so I wanted to fix that. 

First I also had `.. insert/replace into ..`  but I'm not sure that's proper English

Reply:
Hmm, I think I came to the conclusion that insert into somewhat works if we think about this generally as remove & insert operations. We may want to make this more explicit in the process overall.